### ISSUE TITLE
Compile issue and parameters

### ISSUE THREAD

**Issue by xxedocxx** [CityOfZion/neo-python#452]
### Current behavior

- What is the problem?
1. It can't compile when I use boa.interop.Neo.Storage.Find and boa.interop.Neo.Iterator
(I double checked if I missed imports. However, it works with boa compiler.)
<img width="1680" alt="screen shot 2018-06-03 at 6 25 22 am" src="https://user-images.githubusercontent.com/6670368/40887096-05592628-66f8-11e8-9c60-644f1995cbf9.png">

2. Parameter with more than 16 length of array can not be passed.
<img width="1680" alt="screen shot 2018-06-03 at 6 25 57 am" src="https://user-images.githubusercontent.com/6670368/40887091-f247f258-66f7-11e8-9c8b-f03df45d54d1.png">

3. So tried pass integers with bytearray in param but it takes it as string.
<img width="1179" alt="screen shot 2018-06-03 at 6 27 01 am" src="https://user-images.githubusercontent.com/6670368/40887104-33606f54-66f8-11e8-8425-dc64c3a81c36.png">

### Your environment

Let us know in what environment you're running into the issue:

- OS: (Mac os high sierra 10.13.4) 
- neo-python version: neo-python v0.7.0
- Python version: Python 3.6.3

**Comment by brianlenz** [comment]
For item 2, we have a separate issue #213.

I'm not sure about 1 and 3. Do you have the smart contract source code you can share/link to?

**Comment by ediopia** [comment]
I will share it as soon as I finish my current job. Thank you.

**Comment by ixje** [comment]
I can't reproduce `1.` with the latest build
```
neo> build StorageFindTest.py                                                                                                                                                                           
[I 180912 10:40:39 BuildNRun:50] Saved output to StorageFindTest.avm 
```

`2.` is addressed in #213 

`3.` Cannot reproduce

If you think any items in this issue are still applicable please elaborate on your view and provide sample code (where applicable) such that we can reproduce the issue. Thank you
